rclcpp_examples/src/parameters/parameter_events.cpp

```diff
@@ -63,5 +63,7 @@ int main(int argc, char ** argv)
 
   // TODO(wjwwood): Create and use delete_parameter
 
+  rclcpp::spin_some(node);
```

Comment:
I expect this to be a race condition. Not all events might be available yet so potentially only some are handled here. I would suggest to add a sleep before, e.g. 100ms.